frontend/src/BoroughSearch/BoroughSearch.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
```

Comment:
I now see how similar this is to ArtistSearch.tsx - if you still want to combine them I'm all for it